test/OpenTelemetry.Exporter.Zipkin.Tests/ZipkinExporterTests.cs

```diff
@@ -351,14 +351,14 @@ public void ServiceProviderHttpClientFactoryInvoked()
                     break;
                 case StatusCode.Error:
                     statusTag = $@"""{SpanAttributeConstants.StatusCodeKey}"":""ERROR"",";
-                    errorTag = $@",""{ZipkinActivityConversionExtensions.ZipkinErrorFlagTagName}"":""{statusDescription}""";
+                    errorTag = $@"""{ZipkinActivityConversionExtensions.ZipkinErrorFlagTagName}"":""{statusDescription}"",";
```

Comment:
given we are doing this in back-compat way, the existing tests should have required no changes, right?

Reply:
The testcase changes were only reflecting the serializing order of Activity tags which is an *unordered-collection*. 
By adding support for Activity Status, updating the tags order to be - Activity Status or Tag Status is immediately followed by the Activity Error description will have better performance (less unnecessary if/else branching checks.)

I've synced offline with @cijothomas, and we agreed to not modify the testcase in this PR for easier code reviewing experience. And use a follow-up PR to do the performance improvement along with the updated testcase.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/3084